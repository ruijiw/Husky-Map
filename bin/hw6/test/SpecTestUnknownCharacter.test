# Find a path using the LOLDictionary dataSet.
# This test uses both the parser and BFS algorithm in the MarvelPaths
# class and applies to a targeted dataset. This test is for the
# unknown characters case.

LoadGraph dictionary LOLDictionary.tsv

FindPath dictionary TEEMO UNKNOWN

FindPath dictionary UNKNOWN JINX

FindPath dictionary UNKNOWN_A UNKNOWN_B