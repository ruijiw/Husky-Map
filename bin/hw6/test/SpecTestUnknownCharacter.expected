# Find a path using the LOLDictionary dataSet.
# This test uses both the parser and BFS algorithm in the MarvelPaths
# class and applies to a targeted dataset. This test is for the
# unknown characters case.

loaded graph dictionary

unknown character UNKNOWN

unknown character UNKNOWN

unknown character UNKNOWN A
unknown character UNKNOWN B